src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,92 @@
+| Platforms are censorship-resistant and open to everyone.                                                                | Individual access is controlled by centralized entities; platforms can be shut down. 
+| Source code and algorithms are fully available for inspection. Users can vote on platform changes in cases.             | Opaque implementation of algorithms. 
+
+### How do decentralized social networks work? {#decentralized-social-networks-overview}
+Decentralized social networks are a class of [decentralized applications (dApps)](https://ethereum.org/en/developers/docs/dapps/)—applications powered by [smart contracts](https://ethereum.org/en/developers/docs/smart-contracts/) deployed on the blockchain. The contract code serves as the backend for these apps and defines their business logic. 
+
+Traditional social media platforms rely on databases to store user information, program code, and other forms of data. But this creates single points-of-failure and introduces significant risk. For instance, Facebook's servers infamously [went offline for hours](https://www.npr.org/2021/10/05/1043211171/facebook-instagram-whatsapp-outage-business-impact) last year, cutting off users from the platform. 
+
+Decentralized social media website exist on a peer-to-peer network comprising thousands of nodes around the globe. Even if some nodes fail, the network will run uninterrupted, making applications resistant to failures and outages. 
+
+Using decentralized storage systems like the [InterPlanetary File System (IPFS)](https://ipfs.io/), Ethereum-based social networks protect user information from exploitation and malicious use. No one will sell your personal information to advertisers, neither will hackers be able to steal your confidential details. 
+
+Many blockchain-based social platforms have native tokens that power monetization in absence of advertising revenue. Users can buy these tokens to access certain features, complete in-app purchases, or tip their favorite content creators. 
+
+## Decentralized social networks on Ethereum {#Ethereum-social-media}
+
+The Ethereum network has become the preferred tool for developers creating decentralized social media networks owing to the popularity of its tokens (ERC-20/ERC-721) and its massive user base. Here are some examples of Ethereum-based social networks:
+
+- Peepeth
+
+[Peepeth](https://peepeth.com/) is a blockchain-based microblogging site similar to Twitter. The dApp runs on the Ethereum blockchain and uses the InterPlanetary File System (IPFS) to store user data.
+
+Users can send short messages called "Peeps", which cannot be deleted or modified. You can collect tips or tip anyone on the platform in ether (ETH) without leaving the app.
+
+- Mirror
+
+[Mirror](https://mirror.xyz/) is a decentralized and user-owned content distribution platform similar to Medium. Users can sign up by connecting their wallets or providing an Ethereum Name Service (ENS) username.
```

Comment:
Would you consider Mirror to be a social media?

Reply:
"Social platform" would be a good way to describe it. I know the term's a bit loose, but I wanted to include a social platform people were familiar with. I think Mirror's a great example if we want to highlight how Ethereum helps user-owned social communities thrive. 

And IMO, Mirror has more going for it as a social platform than, say, Medium. Asides from Twitter or Discord, Mirror is likely the most popular place for Web3-native brands to connect to communities.

Reply:
Cool 😎 I'm just not very down with the kids in terms of what is/is not considered social media. Agree that Mirror is great to highlight either way.